### PR TITLE
Nominate Eugene Yokota for LSP-WG

### PR DESCRIPTION
**Pull request by eed3si9n**
Could someone nominate me please?

```
scala.NotImplementedError: an implementation is missing
  at scala.Predef$.$qmark$qmark$qmark(Predef.scala:284)
```

### CODE REVIEW

**Comment by xeno-by** [comment]
I can.

File: nominations/eugene-yokota.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Eugene Yokota @eed3si9n
+
+## Proposer
+
+This candidate has been nominated by ???
```
Comment (xeno-by) [inline nominations/eugene-yokota.md]: Please add me here as a proposer.

**Review by xeno-by — CHANGES_REQUESTED** [review]

**Comment by eed3si9n** [comment]
Thanks!

**Comment by adamw** [comment]
I'll be happy to second the nomination

**Comment by eed3si9n** [comment]
Awesome. Thanks guys!

**Pull request merged**